en/peripherals/uavcan_escs.md

```diff
@@ -0,0 +1,98 @@
+# UAVCAN ESCs (Motor Controllers)
+
+PX4 supports the [UAVCAN](https://uavcan.org/) bus for connecting peripherals, including ESCs, GPS modules, various types of sensors, etc.
+
+UAVCAN ESCs have a number of advantages over [PWM ESCs and Servos](../peripherals/pwm_escs_and_servo.md):
+- UAVCAN has been specifically designed to deliver robust and reliable connectivity over relatively large distances.
+  It enables safe use of ESCs on bigger vehicles and communication redundancy.
+- The bus is bi-directional, enabling health monitoring and diagnostics.
+- Wiring is less complicated as you can have a single bus for connecting all your ESCs and other UAVCAN peripherals.
+- Setup is easier as you configure ESC numbering by manually spinning each motor.
+
+PX4 can be used with UAVCAN ESC that run [Sapog](https://github.com/PX4/sapog#px4-sapog) firmware; an advanced open source sensorless PMSM/BLDC motor controller firmware designed for use in propulsion systems of electric unmanned vehicles.
+
+> **Note** At time of writing PX4 supports UAVCAN v0 (not v1.0).
+
+
+## Purchase
+
+Commercially available ESCs running Sapog firmware include:
```

Comment:
Should it not be stated that there are other UAVCAN ESC on the market?

Reply:
@pavel-kirienko  where can we locate the most up-to-date list for market available UAVCAN ESCs?

Reply:
This is the least incomplete list known to me: https://forum.uavcan.org/t/uavcan-esc-options/452/3?u=pavel.kirienko

Reply:
@pavel-kirienko Thanks very much. I have merged this with enough modification to make it clear that there are other options and with a link to your list. 

I'd like to do more, but none of [the links](https://forum.uavcan.org/t/uavcan-esc-options/452/3?u=pavel.kirienko) state support for PX4 or indicate that how they are configured. So from my perspective they aren't really useable.

Can you help me with this? A few questions
1. Is anything in your list other than Zubax Orel 20 and Holybro Koleta20 based on Sapog (ie can be used with same instructions.
2. [Mitochondrik](https://zubax.com/products/mitochondrik) is a chip with UAVCAN interface that is used/integrated in the [Sadulli](https://shop.zubax.com/collections/integrated-drives/products/sadulli-integrated-drive-open-hardware-reference-design-for-mitochondrik?variant=27740841181283) drive. So basically Mitochondric is basically the ESC and is what defines the UAVCAN support for a class of drives (ie it is like Sapog support) - right?
   - How is this used with PX4? Is [connecting](http://docs.px4.io/master/en/peripherals/uavcan_escs.html#connecting) the same for all uavcan systems - e.g. duplicate of sapog docs?
   - How is it enabled on PX4?
   - How is motor enumeration done?
3. Same questions for [Myxa](https://zubax.com/products/myxa) - connection the same? How enable on PX4, how enumerate motors.
4. VESC - same questions. 
5. I can chase up OlliW and NXP directly, but if you have any overview guidance you can share that would be great.